std-bits/table/src/main/java/org/enso/table/read/DelimitedReader.java

```diff
@@ -42,7 +42,9 @@ public class DelimitedReader {
    * <p>I considered to choose `\u0F8EE` which comes from the Private Use Area of the Basic
    * Multilingual Plane. Is has no meaning designated by the Unicode standard.
    */
-  public static final char UNUSED_CHARACTER = '\0';
+  public static final char COMMENT_CHARACTER = '\0';
+
+  public static final char UNUSED_CHARACTER = (char) 63726; // \u0F8EE
```

Comment:
This doesn't work?

```suggestion
  public static final char UNUSED_CHARACTER2 = '\uF8EE';
```

Reply:
This works for me:

```
public class Foo {
  public static final char UNUSED_CHARACTER = (char) 63726; // \u0F8EE
  public static final char UNUSED_CHARACTER2 = '\uF8EE';
  static public void main(String[] args) {
    System.out.println(""+UNUSED_CHARACTER);
    System.out.println(""+UNUSED_CHARACTER2);
  }
}
M:dee 8371-element-direct$ javac Foo.java && java Foo


```

Reply:
It renders like a '[' in my terminal.

Reply:
Oh thank you! I will try. I had the zero in on my machine.